Data/Entities/Note.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace Taskmanager.Data.Entities
+{
+    public partial class Note
+    {
+        public Note()
+        {
+            Todoitems = new HashSet<Todoitem>();
+        }
+
+        public long Id { get; set; }
+        public string Body { get; set; }
+        public long Todoitemid { get; set; }
```

Comment:
I would like to see columns rename in both db and scaffolded code to match PascalCase